src/main/java/talkingbot/task/Deadline.java

```diff
@@ -0,0 +1,56 @@
+package talkingbot.task;
+
+import java.time.LocalDateTime;
+
+import talkingbot.type.TaskType;
+
+/**
+ * A class that represents the Deadline task.
+ */
+public class Deadline extends Task {
+    private final LocalDateTime endTime;
+
+    /**
+     * A constructor for the Deadline class.
+     *
+     * @param description Description of the task.
+     * @param isDone Marks whether the task has been done.
+     * @param endTime Marks the ending time of the deadline.
+     */
+    public Deadline(String description, boolean isDone, String endTime) {
+        super(description, isDone, TaskType.DEADLINE);
+        this.endTime = LocalDateTime.parse(endTime, super.DATE_TIME_ENTRY_FORMAT);
+    }
+
+    /**
+     * Returns the ending time of the Deadline.
+     *
+     * @return Deadline ending time.
+     */
+    public LocalDateTime getEndTime() {
```

Comment:
Should this getter be declared public if it is only used within the class itself?